README.md

```diff
@@ -129,6 +130,7 @@ You can sign up [here](https://join.slack.com/t/rubyforgood/shared_invite/zt-21p
 Please let us know by opening up an issue! We have many new contributors come through and it is likely what you experienced will happen to them as well.
 
 - *"My RBENV installation didn't work!"* - The rbenv repository provides a [rbenv-doctor script](https://github.com/rbenv/rbenv-installer#rbenv-doctor) to verify the installation and check if a ruby version is installed
+- *"I see a database segmentation fault"* - If you're running Apple Silicon, adding `gssencmode: "disable"` to `config/database.yml` should resolve this.
```

Comment:
`database.yml` is checked into git though. Is there a way to solve this that doesn't involve making a file dirty for your own machine?

Reply:
I've tested this on Intel Mac and the option seems okay there too. Assuming it's all okay in Linux too, we could just add it to the `development` section of `database.yml` for everyone, instead of changing the docs?

> `gssencmode`
> This option determines whether or with what priority a secure GSS TCP/IP connection will be negotiated with the server. There are three modes:
>
>   `disable`
>   only try a non-GSSAPI-encrypted connection
> [...]

In development, not encrypting connections seems okay as a default?

Reply:
I think so, yep. I'm good with changing this in the YAML. I'll grab it and see if it causes any problems, and if not we should be good to merge.

Reply:
This is fine on linux as well -- I'd prefer adding it to the database.yml (development and test?) and not mentioning it in the README.